Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -151,6 +151,17 @@ struct PaymentSheetTestPlayground: View {
                         SettingView(setting: $playgroundController.settings.collectPhone)
                         SettingView(setting: $playgroundController.settings.collectAddress)
                     }
+                    Divider().hidden(playgroundController.settings.uiStyle != .embedded)
```

Comment:
ooc can you not put the divider inside the group below?

Reply:
Yeah possible, for some reason we have all our dividers outside their groups 🤔